### ISSUE TITLE
Linux: (Evince) backward search with space in filename not working

### ISSUE THREAD

**Issue by Henridv** [randy3k/LaTeX-Plus#32]
I've removed and reinstalled ST 3 completely to test this.

On Linux (Ubuntu 14.04.2 LTS, 64bit) with the latest ST3 (build 3085), the backward search feature opens different files when the full pathname of the texfiles contains spaces.

e.g.: if the filename of my root file is "/home/username/docs 2014/latex/doc.tex", the backward search from Evince opens the following files in ST3: "/home/username/docs" and "/home/username/docs 2014/latex/2014/latex/doc.tex" or "/2014/latex/doc.tex"

I have the same problem with Latex-Tools, but the same code for backwards search is used, so that was to be expected.
With LaTeXing, I don't have that problem.

**Comment by randy3k** [comment]
Should be fixed now. Unfortunately, this quick fix doesn't allow a filename containing single quote `'`.

**Comment by Henridv** [comment]
Fixed indeed. Thanks for the quick fix!!

Great job on the plugin btw!

**Comment by me10240** [comment]
I wanted to add this observation. I just switched to Ubuntu 14.04 and on ST3, I found after  the installation that forward search works but inverse doesnt. Name of the executable was correct, none of the usual suspects. 
Finally, the issue  was that  **I had not set up an ST3 launcher for the dashboard  by creating the appropriate sublime.desktop file**  Once I did that inverse search worked perfectly.
